src/OrchardCore/OrchardCore.ContentManagement/DefaultContentManager.cs

```diff
@@ -388,14 +388,7 @@ public async Task CreateAsync(ContentItem contentItem, VersionOptions options, b
 
             await ReversedHandlers.InvokeAsync((handler, context) => handler.CreatedAsync(context), context, _logger);
 
-            if (!invokeUpdateCallbacks)
-            {
-                _session.Save(contentItem);
-            }
-            else
-            {
-                await UpdateAsync(contentItem);
-            }
+            await UpdateAsync(contentItem);
```

Comment:
Need to test but it should work as fine as it will also double call _session.Save(contentItem).

Reply:
Actually why calling the Update events when we are doing a Create. I'm reverting to `_session.Save(contentItem)`. Will do test and report what I find. So far by using ContentCreateTask with a workflow everything works fine.